eHealthAfrica/couchdb-user-management-app
# Men 657

### What does this PR do?
* Updates the UMS to no longer store the contents of the 'workspace' in couchdb upon entry update

### How should this be manually tested or reproduced?
1. Launch the dashboard and login as an admin
2. Navigate to the UMS
3. Modify any user
4. Observe the datastructure and ensure that it has not been assigned a new property called umsWorkdpave

PS: if the property already exists (from the previous bug), this would not remove it.

### Any background context you want to provide?
 None

### What are the relevant JIRA stories?
https://jira.ehealthafrica.org/browse/MEN-657

### What are the impacted areas in the application?
  None

### Any related PR
    None

### Screenshots (if appropriate)
    None
